config/environments/development.rb

```diff
@@ -10,9 +10,14 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
   # config.action_mailer.delivery_method = :sendmail
-  host = "localhost:3000"
-  config.action_mailer.default_url_options = { host: host }
-
+  config.action_mailer.smtp_settings = {
+    :address   => "smtp.mandrillapp.com",
+    :port      => 25, # ports 587 and 2525 are also supported with STARTTLS
+    :enable_starttls_auto => true, # detects and uses STARTTLS
+    :user_name => "michael.dao@gmail.com",
+    :password  => ENV["PIVOT_MANDRILL"], # smtp password is any valid api key
+    :authentication => 'login' # Mandrill supports 'plain' or 'login'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.